axonserver/src/main/java/io/axoniq/axonserver/config/MessagingPlatformConfiguration.java

```diff
@@ -215,6 +259,19 @@ public void setHostname(String hostname) {
     }
 
     public String getDomain() {
+        if (domain == null) {
```

Comment:
I would not add this logic. It may be that a hostname returns the internal FQDN, and this sets the external domain.